src/content/tools/en/across.mdx

```diff
@@ -0,0 +1,20 @@
+---
+name: "Across"
+category: ["Bridge"]
+excerpt: ""
```

Comment:
I'd recommend adding in an excerpt and also some more details on Across in the body so devs have a better understanding of what it is